spec/helpers/bootstrap.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var fs = require('fs-plus');
+var path = require('path');
+var remote = require('remote');
+var app = remote.require('app');
+var pkgJson = require('../../package.json');
+var newLine = require('os').EOL;
+
+// Start the crash reporter before anything else
+require('crash-reporter').start({
+  productName: pkgJson.name,
+  companyName: 'atom-shell-starter'
+});
+
+require('node-jsx').install({ extension: '.jsx', harmony: true });
```

Comment:
on #44 @mfunkie mentioned that `node-jsx` is fading away along with the deprecated [JSTransform](http://facebook.github.io/react/blog/2015/06/12/deprecating-jstransform-and-react-tools.html).

`electron-compile` should contain **babel** and whatever else is needed to run `jsx` and `es6`